src/views/product-downloads-view/components/release-information/release-information.module.css

```diff
@@ -29,6 +29,10 @@
 
 .armUserNote {
 	padding: 16px;
+
+	& span:last-of-type {
+		width: 100%;
+	}
```

Comment:
These styles were added so the code blocks on the consul install page are full width like the previous version.